packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/data_source/pin_biometrics_local_data_source.dart

```diff
@@ -6,41 +6,33 @@ import '../../base/data_sources/local/shared_preferences_instance.dart';
 /// You have to implement and provide a [BiometricsLocalDataSource], you can
 /// store the value of [_areBiometricsEnabled], for example in [SharedPreferences]
 class PinBiometricsLocalDataSource implements BiometricsLocalDataSource {
-  PinBiometricsLocalDataSource(
-    this._sharedPreferences,
-  );
+  PinBiometricsLocalDataSource(this._sharedPreferences);
 
-  bool? onRestart;
   final SharedPreferencesInstance _sharedPreferences;
 
   /// This value is the state for the user general choice for the app
   static const _areBiometricsEnabled = 'areBiometricsEnabled';
 
-  /// This value is changed in while using the app to stop and start auto biometrics
-  static const _areBiometricsEnabledWhileUsingTheApp =
-      'areBiometricsEnabledWhileUsingTheApp';
+  /// This value is changed, while using the app to stop and start automatic
+  /// biometrics authentication
+  bool? _disabled = false;
```

Comment:
Any reason for this _disabled to be nullable?

Reply:
No, it was changed